onnxruntime/core/providers/cpu/tensor/slice_helper.h

```diff
@@ -136,20 +136,12 @@ inline Status PrepareForComputeHelper(const gsl::span<const int64_t>& raw_starts
 
     // process end
     auto end = raw_ends[axis_index];
-    // INT_MAX has a special meaning for end according to spec
-    // equivalent to 'None' in numpy
-    // it represent slicing to the end of the dimension
-    if (end == std::numeric_limits<int32_t>::max() ||
-        end == std::numeric_limits<int64_t>::max()) {
-      end = step < 0 ? -1 : dim_value;
```

Comment:
Removed this special-casing

Reply:
If I have a model with opset 10 and was using INT_MAX as the special token, does that model still work?